### DISCUSSION TITLE
Further tray icons improvement

### DISCUSSION THREAD

**Discussion by ionGL** [Rem0o/FanControl.Releases#1385] [Ideas]
This multiple tray icons feature was much needed!

A few things could be improved further:

- Have an option to select a custom color for each icon. (argus has this)
<a href='https://postimages.org/' target='_blank'><img src='https://i.postimg.cc/CzGwBnMw/argus.jpg' border='0' alt='argus'/></a>

- An option to change the tray icons font. If not individually, all at once. Size, bold etc.

- Mouse-over could be better. Maybe on 2-3 rows? .json name plus the program version might not be needed there. Maybe add the CPU/GPU etc name as they appear in the "Add tray icon sensor" dropdown?
- <a href='https://postimages.org/' target='_blank'><img src='https://i.postimg.cc/SRRQTKdG/fancontroltray.jpg' border='0' alt='fancontroltray'/></a>

**Comment by Huddling1705** [comment]
Custom colors are really needed. Multiple Temps in tray are not really usable without colors.
so long i will use afterburner for this

**Reply by Rem0o** [reply]
It is coming, releasing features as they are ready. Could have waited, but I prefer shipping quickly as soon as a minimal working version is ready.

**Reply by Huddling1705** [reply]
awesome. Thank you very much, I really appreciate your work :)

**Reply by Bhaalspawn** [reply]
> It is coming, releasing features as they are ready. Could have waited, but I prefer shipping quickly as soon as a minimal working version is ready.

Thank for you software, i really like it. I just discovered it
I still have to test more deeply but i don't need so many features. i already set custom curves and it seems it's working. My only request was going to be about sensor icons, that's why i'm using this thread
I see that you already added custom colors for text, but what about a color for background like AIDA?

**Comment by ionGL** [comment]
Awesome development with the introduction of custom tray icon color!

Now we only miss custom fonts and a better organization of text for mouse-over!

**Reply by Rem0o** [reply]
See V165

**Reply by ionGL** [reply]
Just tested and it the tooltip text on mouse hover looks nice!

Is custom font a possibility? If not, maybe size, bold on/off etc?
